extensions/amp-analytics/0.1/config.js

```diff
@@ -490,3 +491,24 @@ function expandRequestStr(request) {
     'baseUrl': request,
   };
 }
+
+/**
+ * Handles top level fields in the given config
+ * @param {!JsonObject} config
+ * @return {JsonObject}
+ */
+function handleTopLevelAttributes_(config) {
+  // handle a top level requestOrigin
+  if (hasOwn(config, 'requests') && hasOwn(config, 'requestOrigin')) {
+    const requestOrigin = config['requestOrigin'];
+
+    for (const requestName in config['requests']) {
+      // only add top level request origin into request if it doesn't have one
+      if (!hasOwn(config['requests'][requestName], 'origin')) {
```

Comment:
Can we add test coverage to `requestOrigin` and `origin` please. Thank you

Reply:
Done!